### DISCUSSION TITLE
Sync just CSS? (from config)

### DISCUSSION THREAD

**Discussion by minchinweb** [remotely-save/remotely-save#130] [Ideas]
I was wondering how involved it would be to make it so I could sync the CSS snippets of the .obsidian folder, while leaving the rest of the folder alone?

I can see syncing the .obsidian folder between clients could cause issues, but moving around the CSS snippets should be safe.

Thanks for making the plugins; it is much appreciated!

**Comment by fyears** [comment]
Where are the CSS snippets located? Currently no way to do that but the feature may or may not be added in the future releases.

**Reply by minchinweb** [reply]
`[vault]\.obsidian\snippets`

**Reply by minchinweb** [reply]
(I don't think the folder is created by default)

**Comment by fyears** [comment]
the latest version allows setting deny list so you may be able to block all other sub folders in .obsidian
